docs/settings/canned-message-plugin.md

```diff
@@ -128,13 +131,12 @@ Configuring this setting is not yet available for the selected platform. If this
   </TabItem>
 </Tabs>
 
-### canned_message_plugin_messages
-
-Predefined messages for CannedMessagePlugin separated by `|` characters.
+### canned_message_plugin_send_bell
 
-You can define up to 50 messages with a total length 1024 bytes.
+CannedMessagePlugin also sends a bell character with the messages.
```

Comment:
"an ASCII bell character"

There's a chance a user hasn't ever seen this before and doesn't know what it is, so a little more specific might be helpful.